docs/nimservice.md

```diff
@@ -72,6 +72,8 @@ meta-llama3-8b-instruct-job-xktnk          0/1     Completed   0          4m38s
 
 ### 3. Verify the Microservice is Running
 
+#### Example 1:
```

Comment:
Consider indicating when someone would want to run the first one vs the second one.

Reply:
Add the example name

Reply:
Yes, it would be good to indicate the example based on use case. @angudadevops you can refer here: https://docs.nvidia.com/nim/large-language-models/latest/getting-started.html#openai-completion-request

Reply:
Updated the names accordingly

Reply:
lgtm, we can iterate further on docs. @mikemckiernan please review if this looks good to you.